tests/cypress/integration/remove_users_tasks.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+let authKey = ''
+
+describe('Delete users and tasks created during the test run.', () => {
+    it('Get token', () => {
+        cy.request({
+            method: 'POST',
+            url: '/api/v1/auth/login',
+            body: {
+                username: Cypress.env('user'),
+                password: Cypress.env('password')
+            }
+        })
+        .then((responce) => {
+            authKey = responce['body']['key']
+        })
+    })
+    it('Get a list of users and delete all except id:1', () => {
+        cy.request({
+            url: '/api/v1/users',
+            headers: {
+                Authorization: `Token ${authKey}`
+            }
+        })
+        .then((responce) => {
+            const responceResult = responce['body']['results']
+            for (let user of responceResult) {
+                let userId = user['id']
+                if (userId !== 1) {
+                    cy.request({
+                        method: 'DELETE',
+                        url: `/api/v1/users/${userId}`,
+                        headers: {
+                            Authorization: `Token ${authKey}`
+                        }
+                    })
+                }
+            }
+        })
+    })
+    it('Get a list of tasks and delete them all', ()=> {
+        cy.request({
+            url: '/api/v1/tasks',
+            headers: {
+                Authorization: `Token ${authKey}`
+            }
+        })
```

Comment:
Can we use ``cvat-core`` instead of these direct requests here and above?
For example to get task list you can use ``cvat.tasks.get(...)``.
What's more as far as I remember, this request doesn't return all task (only first 10). Then need to use parameter ``page`` to get more

One thing ``cvat-core`` isn't able to do is to delete a user. We can leave this case as is, or I can add api method for this. We can have a call and discuss it

Reply:
In the current moment, as we agreed in discuss, I changed the link to search for tasks by adding `?page_size=1000`